meilisearch-http/src/index_controller/update_actor/update_store.rs

```diff
@@ -13,16 +13,16 @@ use uuid::Uuid;
 use crate::helpers::EnvSizer;
 use crate::index_controller::updates::*;
 
-type BEU64 = heed::zerocopy::U64<heed::byteorder::BE>;
+type Beu64 = heed::zerocopy::U64<heed::byteorder::BE>;
```

Comment:
This name should remain like this. add `#[allow(non_camel_case)]` on top of it should fix it

Reply:
Not pb, I change this. By curiosity, could you quickly explain why? 🙂

Reply:
this is an acronym for Big Endian U64, so it makes more sense that is is all uppercased
This is also how we refer to this type in milli, and meilisearch :)